target-osgi-environment/maximum-environment/pom.xml

```diff
@@ -0,0 +1,229 @@
+<?xml version="1.0"?><!--
```

Comment:
I'm worried about maintaining this - this is a big moving target and you cannot use version numbers that are not released yet... while if you are just "overly confident" in the `Import-Package` statement and add something like even `version="[0,4)` (at the moment we have `version="[0,3)`), then it'll "just work" for a long time without changes (it'll break at runtime for breaking changes obviously)

But in the end what counts is version range you give and this only counts at build time - so I suppose a good addition ("just a tool") to be able to test a version range without having to spin up an exact AEM version with a given SP for this.

Reply:
There is no need for maintenance except if you increase the minimum requirement